### PR TITLE
Paulo Barata

### PR DESCRIPTION
**Pull request by PBara7a**

### CODE REVIEW

File: src/poker.js [hunk 1]
```diff
@@ -1,5 +1,63 @@
...
+  }
+
+  findBest (arr) {
+    let best = arr[0]
+    for (let i = 1; i < arr.length; i++) {
+      if (scoreBoard[arr[i][0]] > scoreBoard[best[0]]) best = arr[i]
+    }
+    return best
+  }
+}
```
Comment (d-username) [inline src/poker.js:62]: I am going to put this comment on the whole code. It is really good and compact. I don't understand every part of it how it works but I see that you created some reusable methods for the extension criteria.

Comment (Dan2024) [reply src/poker.js:62]: Similar to David, it's really compact and readable - the descriptive functions really help with this. The use of the spread/rest operator here is really nice too and I'd be interested to understand exactly how the findBest function works 👍

File: src/poker.js [hunk 2]
```diff
@@ -1,5 +1,63 @@
-class Poker {
-  // Write your implementation here
+const scoreBoard = {
+  1: 1,
```
Comment (Alex90Jennings) [inline src/poker.js:2]: never seen a 1 card ;-)

Comment (PBara7a) [reply src/poker.js:2]: Just noticed this now as well ahah -.-' don't tell me how to run my casino sir.

**Pull request closed without merging**